### ISSUE TITLE
encounter error when unwrapping post-trained model

### ISSUE THREAD

**Issue by yfpeng1234** [thuml/RLVR-World#2]
Hi @Manchery ,
Thanks for providing the code. I try RLVR pos-training for video prediction model. I want to evaluate the post-trained model on test set for different training steps. The intermediate ckpts look like this:

![Image](https://github.com/user-attachments/assets/af02c0d0-257d-4f84-a7b7-1b6c973c0b12)
Then I try to evaluate model trained for 10 steps by:
`export CUDA_VISIBLE_DEVICES=3
python eval_vgpt_multiturn.py --per_device_eval_batch_size 4 \
    --config_name configs/vgpt/ctx_llama_small.json\
    --dataset_path $DATASET_PATH \
    --pretrained_model_name_or_path $PRETRAINED_MODEL_PATH/rt1-compressive-tokenizer \
    --pretrained_transformer_path $VERL_MODEL_PATH/verl_grpo_example_vgpt/ctx_vgpt_msp8/global_step_10/actor \
    --processor_type ctx_msp \
    --output_jsonl eval_jsonl/vgpt_small_ctx_msp8_head12_fulleval_release_10step.jsonl \
    --max_decode_batchsize 1 \
    --segment_length 8 \
    --use_eval_dataset \
    --max_eval_iters 400 \
    --exp_name vgpt_small_ctx_msp8_head12_fulleval_10step $@`
But I encountered the following error, which I think should be related to model unwrapping:

![Image](https://github.com/user-attachments/assets/d20f2d8e-af93-4216-af1e-3d72b2fd99b4)

**Comment by Manchery** [comment]
Thanks for your interest in our work!!!

It seems like you need the following script to transform models saved by verl.

```
python merge_sharded_ckpts.py --ckpt_path xxxxxx/actor --config_path {path to the pretrained model for providing configs}
```

Then you can see an unwrapped model under the path xxxxxx/actor.

Hope this information can help you fix the problem. I apologize for being a bit busy this week and will review this part of the code and provide detailed instructions in README, as soon as possible.

**Comment by yfpeng1234** [comment]
Thanks a lot for your prompt reply. I will try that!
